Cargo.toml

```diff
@@ -17,7 +17,7 @@ cc = "1.0"
 [dependencies]
 lalrpop-util = "0.19"
 regex = "1"
-num-bigint = "0.4"
+num-bigint = "0.4.0"
```

Comment:
Why is this changed?

Reply:
I wanted to have something that would match precisely two versions of num_bigint I have. But I would try out returning it to previous value to see if it works well.